lighthouse-core/gather/gatherers/dobetterweb/tags-blocking-first-paint.js

```diff
@@ -27,12 +27,13 @@ const Gatherer = require('../gatherer');
 function collectTagsThatBlockFirstPaint() {
   return new Promise((resolve, reject) => {
     try {
-      const tagList = [...document.querySelectorAll('link, head script[src]')]
+      const tagList = [...document.querySelectorAll('link, head script[src], script[type]')]
```

Comment:
Why was this added?

Reply:
To detect the inline modules, type="module" may not be always followed by the `src` attribute.

```html
<script type="module">
import module from '../example.js';
module.test();
</script>
```

Reply:
sure, but I mean we're trying to ignore modules so we'd be pulling them in just to ignore them